pkg/distro/validator_test.go

```diff
@@ -0,0 +1,1196 @@
+package distro_test
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/osbuild/images/internal/common"
+	"github.com/osbuild/images/pkg/blueprint"
+	"github.com/osbuild/images/pkg/disk"
+	"github.com/osbuild/images/pkg/distro"
+	"github.com/osbuild/images/pkg/manifest"
+	"github.com/osbuild/images/pkg/platform"
+	"github.com/osbuild/images/pkg/rpmmd"
+	"github.com/stretchr/testify/assert"
+)
+
+type TestImageType struct {
```

Comment:
(nitpick) I would love if we had only one TestImageType, ideally from `test_distro` and with exported fields so that each test that needs an ImageType can just import and tweak it. But I understand its probably a bit cumbersome right now with the existing `test_distro`

Reply:
Can we move this to a follow-up?  We could also solve this by breaking down the image type interface, which I would like to do at some point anyway.